agent/lib/kontena/service_pods/infra_manager.rb

```diff
@@ -0,0 +1,76 @@
+module Kontena
+  module ServicePods
+    class InfraManager
```

Comment:
The `Manager` somehow made me assume this was an Actor.

Reply:
Yeah, probably bad name.